sdk/operationalinsights/Microsoft.Azure.OperationalInsights/src/Microsoft.Azure.OperationalInsights.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Microsoft Azure Operational Insights Data Library</Description>
+    <Description>Provides Microsoft Azure Operational Insights data operations</Description>
+    <AssemblyTitle>Microsoft Azure Operational Insights</AssemblyTitle>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```

Comment:
As we chatted please add the version. Looking at the old project it seems as though we are missing the following:
```
    <VersionPrefix>0.10.0-preview</VersionPrefix>
    <PackageTags>Microsoft Azure OperationalInsights;OperationalInsights</PackageTags>    
	  <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
```